### ISSUE TITLE
Enhancement Request: Output the buffer

### ISSUE THREAD

**Issue by jmorris644** [bartbutenaers/node-red-contrib-plate-recognizer#5]
Is it possible to also output the incoming buffer as part of the JSON output string? I need to do some follow-on image manipulation once I have the JSON info. For example, I want to crop the original image to just save the bounding box of the vehicle.

Optionally it would be nice to output the cropped vehicle image. This would be similar to what the current Stream SDK does.

**Comment by bartbutenaers** [comment]
Hi @jmorris644,
I see now that I have not described this on my readme page.  The input message is forwarded to the output. 
 Suppose your image buffer arrives in the input image as `msg.payload` and your recognition result is stored (via [output field](https://github.com/bartbutenaers/node-red-contrib-plate-recognizer#output-field) also in `msg.payload`.  Then your image in the message will be overwritten by your recognition results.  But if you store e.g. your recognition results in `msg.myresults`, then your `msg.payload` should still contain your original input image...
Bart

**Comment by jmorris644** [comment]
That makes perfect sense.

**Comment by bartbutenaers** [comment]
> Optionally it would be nice to output the cropped vehicle image. 

Yes indeed that can be useful.  But as a separate node, so it can be reused to crop the image output of other nodes also.  Will put it on my todo list...

**Comment by jmorris644** [comment]
I am currently using node-red-contrib-image-tools to do the cropping.
